src/css/style.css

```diff
@@ -104,3 +109,80 @@ img {
 .card:hover {
     transform: scale(1.1)
 }
+
+
+
+/* code for service_cards page */
+
+.services_heading {
+display: flex;
+justify-content: center;
+font-weight: bolder;
+font-size: xx-large;
+margin-top: 40px;
+margin-bottom: 40px;
+}
+
+.servicesBox {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;}
+
+
+.servicesPicture {
+    width: 100%;
+    /* border-radius: 12px; */
+    height: 200px;
+    object-fit: cover;
+}
+
+.service_card_container {
+    width: 375px;
+    max-height: 600px;
+    margin: 15px;
+    margin-top: 10px;
+    transition: 0.2s;
+    padding: 10px;
+}
+
+.info_card {
+    background-color: rgba(158, 163, 168, 0.692);
+    border-radius: 6px;
+    overflow: hidden;
+    box-shadow: 1px 1px 6px black;
+}
+
+ul.service_lists {
+    list-style-type: disc;
+    max-width: 300px;
+    max-height: 600px;
+    margin: 30px;
+    margin-top: 10px;
```

Comment:
like I mentioned in the heading, use padding on the container rather than margin on the elements to create space between the container edges and the inside elements. You probably just want a margin top here to add space between this and the heading.

Reply:
I need assistance with this one